ocfweb/docs/docs/communications/stf-renewal.md

```diff
@@ -0,0 +1,25 @@
+[[!meta title="Upcoming Student Technology Fund renewal voting"]]
```

Comment:
Upcoming Student Technology Fund Renewal Vote 

or at least capitalize all the words